05/app.js

```diff
@@ -11,26 +11,40 @@ const stats = {
 
 const pElements = document.querySelectorAll("p");
 const aElement = document.querySelector("a");
-let value = 0;
-let valueLink = 0;
 
 const clickCounter = function (e) {
   e.preventDefault();
-  if (e.target.tagName === "P" && e.target.getAttribute("data-id") === "p1") {
-    stats["paragraphs"]["p1"] = value;
-    value++;
+  const attr = e.target.getAttribute("data-id");
+  const linkHref = e.target.getAttribute("href");
+
+  if (typeof stats.paragraphs[attr] == "null") {
+    console.log("null");
+  }
+  if (typeof stats.links[linkHref] === "null") {
+    stats.links[linkHref] = "";
+  }
+  if (
```

Comment:
Raczej bym zrobił najpierw `if` sprawdzający czy to `P` lub `A`, a potem wykonywał działania. Będzie prostsza logika:
```
if(Jeśli e.target to P) {
    if(Jeśli nie ma rozpoczątego naliczania) {
        ustaw wartość na 1
    } else {
        zwiększ wartość o 1
    }
} else {
    ...
}
```